rubin_scheduler/scheduler/surveys/surveys.py

```diff
@@ -35,6 +35,7 @@ def __init__(
         camera="LSST",
         area_required=None,
         fields=None,
+        **kwargs,
```

Comment:
I guess since the BaseSurvey does not accept random kwargs, this is probably ok now, but this is how we ended up with random keyword arguments that did nothing yet were expected to do things.